arch/arm/core/Kconfig

```diff
@@ -42,6 +42,19 @@ config ARM_STACK_PROTECTION
 	  This option enables MPU stack guard to cause a system fatal error
 	  if the bounds of the current process stack are overflowed.
 
+config ARM_SECURE_FIRMWARE
+	bool
+	default n
+	help
+	  This option indicates that we are building a Secure firmware for
```

Comment:
I am not convinced about using firmware as a countable noun. I don't think one can say "one firmware and two firmwares". I feel we need to clarify this further.